utility/src/test/java/org/oppia/android/util/networking/BUILD.bazel

```
@@ -0,0 +1,91 @@
+"""
+Tests for networking-specific test utilities.
```

```suggestion
Tests for networking-specific utilities.
```

This tests production utilities, too.

---

Done.